linux-tools/augeas.py

```diff
@@ -0,0 +1,52 @@
+#!/bin/python
+import os, subprocess
+import logging
+
+from autotest.client import test
+from autotest.client.shared import error
+
+class augeas(test.test):
+
```

Comment:
The path of augeas.py is wrong. It should be under linux-tools/augeas. Please correct it.